sale_order_import/wizard/sale_order_import.py

```diff
@@ -470,6 +470,13 @@ def _prepare_create_order_line(
             vals["order_id"] = order
             vals = solo.play_onchanges(vals, ["product_id"])
             vals.pop("order_id")
+
+        # Handle additional fields dynamically if available.
+        # This way, if you add a field to a record and it's value is injected by a parser
+        # you won't have to override `_prepare_create_order_line` to let it propagate.
+        for k, v in import_line.items():
+            if k not in vals and k in solo._fields:
+                vals[k] = v
```

Comment:
```suggestion
        for field in import_line:
            if field not in vals and field in solo._fields:
                vals[field] = import_line[field]
```

Reply:
I don't get how this is better? :thinking: